lib/smack/SmackRep.cpp

```diff
@@ -5,7 +5,11 @@
 #include "smack/SmackRep.h"
 #include "smack/SmackOptions.h"
 
-#include <iostream>
+#ifdef __x86_64__ 
```

Comment:
This should be extracted from the the program or target somehow. Perhaps just looking at the parameter type of `malloc` would be a good option.